docs/index.md

```diff
@@ -1584,7 +1584,7 @@ Environment variables and default values:
 | `--vmwarevcloudair-docker-port`   | `VCLOUDAIR_DOCKER_PORT`   | `2376`                                     |
 
 #### VMware vSphere
-Creates machines on a [VMware vSphere](http://www.vmware.com/products/vsphere) Virtual Infrastructure. Requires a working vSphere (ESXi and optionally vCenter) installation. The vSphere driver depends on [`govc`](https://github.com/vmware/govmomi/tree/master/govc) (must be in path) and has been tested with [vmware/govmomi@`c848630`](https://github.com/vmware/govmomi/commit/c8486300bfe19427e4f3226e3b3eac067717ef17).
+Creates machines on a [VMware vSphere](http://www.vmware.com/products/vsphere) Virtual Infrastructure. Requires a working vSphere (ESXi with non free license or 60 days trial and optionally vCenter) installation. The vSphere driver depends on [`govc`](https://github.com/vmware/govmomi/tree/master/govc) (must be in path) and has been tested with [vmware/govmomi@`c848630`](https://github.com/vmware/govmomi/commit/c8486300bfe19427e4f3226e3b3eac067717ef17).
```

Comment:
Hi @cdosso thank you for the contribution.  You have a lot packed into that sentence.  If we unpack it, hopefully we can clarify.  In researching, I referenced http://www.mustbegeek.com/difference-between-vsphere-esxi-and-vcenter/

**Suggested alternate:**

The machine must have a working vSphere ESXi  installation.  You can use a paid license or free 60 day trial license. Your installation may also include an optional VCenter server.